src/components/AddTodo/index.ts

```diff
@@ -0,0 +1,2 @@
+import AddTodo from './AddTodo';
```

Comment:
Not particularly useful file. I see what you wanted to do here, but this pattern creates more problems than it solves. For example:

- You might unintentionally cause [circular dependencies](https://medium.com/visual-development/how-to-fix-nasty-circular-dependency-issues-once-and-for-all-in-javascript-typescript-a04c987cf0de);
- When you review `AddTodo` file, it won't be marked as unused by IDE, even if it's actually not used anywhere. This is just because it's imported here;
- If you are to implement lazy-loading later on, this pattern will really fuck up load order.